src/components/Footer.js

```diff
@@ -20,7 +22,7 @@ const Footer = () => {
         inverted
         stackable>
         <Grid.Row>
-          <Grid.Column width={ 7 }>
+          <Grid.Column width={ 4 } floated='left' left>
```

Comment:
The attribute `left` is causing errors. I believe it can just be removed.

Reply:
(They show up in the browser console, not in the terminal)

Reply:
The left thing is placed in there to keep the styling from Semantic UI React. I dunno at least that's why I did it that way without using custom CSS. Unless we prefer to use custom CSS I'll try to do that instead.

Reply:
Really? When I removed it, `floated='left'` seemed to be sufficient. Maybe I'm wrong, though. I've been pretty tired.

Reply:
Oh hmm, maybe just the left at the end is needed then. Sounds good! I'll remove it.